packages/teleport/src/Console/DocumentSsh/useSshSession.ts

```diff
@@ -22,14 +22,50 @@ import Tty from 'teleport/lib/term/tty';
 import ConsoleContext from 'teleport/Console/consoleContext';
 import { useConsoleContext } from 'teleport/Console/consoleContextProvider';
 import { DocumentSsh } from 'teleport/Console/stores';
+import {
+  makeMfaAuthenticateChallenge,
+  makeWebauthnAssertionResponse,
+} from 'teleport/services/auth';
 
 export default function useSshSession(doc: DocumentSsh) {
   const { clusterId, sid, serverId, login } = doc;
   const ctx = useConsoleContext();
   const ttyRef = React.useRef<Tty>(null);
+  const tty = ttyRef.current as ReturnType<typeof ctx.createTty>;
   const [session, setSession] = React.useState<Session>(null);
   const [statusText, setStatusText] = React.useState('');
   const [status, setStatus] = React.useState<Status>('loading');
+  const [showReauthnDialog, setShowReauthnDialog] = React.useState(false);
```

Comment:
`showReauthnDialog` sounds as a verb whereis it's not, instead -> `isAuthnDialogVisible`